pkg/squid/db/migrations/1656927980028-addWebToIdentity.js

```diff
@@ -1,11 +0,0 @@
-module.exports = class addWebToIdentity1656927980028 {
-  name = 'addWebToIdentity1656927980028'
-
-  async up(db) {
```

Comment:
`web` existet before,
`web3name` and `discord` have been added. or is this an older migration?

Reply:
I dropped all of the old migrations, and it's a new one. My branch was outdated.

Reply:
how am I supposed to know this? 🗡️